dashboard/app/controllers/pd/mail_preview_controller.rb

```diff
@@ -152,9 +152,17 @@ def enrollment(workshop)
       workshop: workshop,
       name: 'Tracy Teacher',
       email: 'tracy_teacher@example.net',
+      school_district: school_district,
+      school: 'sample school'
     ).tap(&:assign_code)
   end
 
+  def school_district
+    SchoolDistrict.new(
+      name: 'sample district'
```

Comment:
yeah I'm not saving it, just passing the partially-filled object to the preview.